src/router/routes.ts

```diff
@@ -3,36 +3,59 @@
 // import StateExample from '@/views/StateExample.vue'
 import DefineCompany from '@/views/DefineCompany.vue'
 import ReviewConfirm from '@/views/ReviewConfirm.vue'
+import SigninView from '@/views/auth/Signin.vue'
+import SignoutView from '@/views/auth/Signout.vue'
 
 export const routes = [
   {
     path: '/',
     redirect: '/define-company'
   },
-  // {
-  //   path: '/home',
-  //   name: 'home',
-  //   component: Home,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/MixinExample',
-  //   name: 'mixinExample',
-  //   component: MixinExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/StateExample',
-  //   name: 'stateExample',
-  //   component: StateExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
+  {
+    path: '/signin/:idpHint',
+    name: 'signin',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl',
+    name: 'signin-redirect',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl/:redirectUrlLoginFail',
+    name: 'signin-redirect-fail',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signout',
```

Comment:
How do we get to this route? From the header component? May need to check with Andrew for this.

Reply:
This route is set by the SbcHeader component ... but please add a code comment so this makes sense to the next developer.

Reply:
Someone can add such a comment in a future commit (since this code is subject to change anyway with the NR query and draft resuming that are also being implemented at the moment).

Reply:
Done.